content/en/account_management/billing/custom_metrics.md

```diff
@@ -7,64 +7,206 @@ aliases:
+* Adding tags on a metric can change the number of custom metrics (number of unique tag value combinations) associated with that particular metric -- see Example 3 and 4 below.
+* Reordering of tag values doesn’t add uniqueness, the following combinations are the same custom metric:
+
+  *  `metric_name{tag_1:value_1, tag_2:value_2}`
+  *  `metric_name{tag_2:value_2, tag_1:value_1}`
+
+Find below some example of how to count your custom metrics, depending on your [submision type][8], the amount of custom metrics generated may difer:
+
+{{< tabs >}}
+{{% tab "Count, Rate, Gauge" %}}
+
+**Note**: This example uses a [`COUNT` metric type][1], but the count of custom metrics would have been the same with a [`GAUGE`][2] or a [`RATE`][3] metric type.
+
+Suppose you’re submitting a metric, `request.Count`, from two hosts (A,B), which counts the number of endpoint requests. You’re submitting this metric with two tag keys:
+
+* `endpoint`  that can take the value `X` or `Y`
+* `status` that can take the value `200` or `400`
+
+Let’s assume that in your data, endpoint `X` is supported by both hosts, but fails only on `host:B`, and requests to `endpoint:Y` are always successful and only appears on `host:B` as shown below:
+
+{{< img src="account_management/billing/custom_metrics/custom_metrics_host.png" alt="Custom metrics host" responsive="true" style="width:80%;">}}
+
+There are then **4 Custom metrics**, one for each unique combinations of metric name, host, and tag values for your `request.Count` metric. The custom metrics are listed below:
+
+| Metric Name     | Tag Values                           |
+| --------------- | ------------------------------------ |
+| `request.Count` | `host:A`, `endpoint:X`, `status:200` |
+| `request.Count` | `host:B`, `endpoint:X`, `status:200` |
+| `request.Count` | `host:B`, `endpoint:X`, `status:400` |
+| `request.Count` | `host:B`, `endpoint:Y`, `status:200` |
+
+[1]: /developers/metrics/types/?tab=count
+[2]: /developers/metrics/types/?tab=gauge
+[3]: /developers/metrics/types/?tab=rate
+{{% /tab %}}
+{{% tab "Histogram" %}}
+
+**A HISTOGRAM metric generates five custom metrics** for each unique combination of metric name, host, and tag values to support the Agent-side aggregations: `count`, `sum`, `min`, `max`, and `avg`. [Learn more about HISTOGRAM metric type][1].
+
+Suppose you have a histogram metric measuring `request.Latency` for two endpoints `X` and `Y`  on a single host:
+
+|| INSERT IMAGE  ||
+
+The number of unique tag value combinations submitted for a `HISTOGRAM` metric with this tagging scheme is 2, but the Agent generates [5 custom metrics][1] for each of the original 2 unique tag value combinations to account for each Agent-side aggregations: `count`, `sum`, `min`, `max`, and `avg`. This results in a total of 10 custom metrics for this `HISTOGRAM` metric: :
+
+| Metric Name       | Tag Values             | Aggregations                            |
```

Comment:
These aggregations are incorrect (See https://docs-staging.datadoghq.com/gus/custom-metrics/developers/metrics/types/?tab=histogram) 

 Users can also change their config yaml file so it may not always be 5...